examples/ext_authz/run_envoy.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/local/bin/envoy -c "/etc/envoy-${FRONT_ENVOY_YAML}" --service-cluster front-proxy
```

Comment:
Should be `/etc/envoy-config/${FRONT_ENVOY_YAML}`?

Reply:
no - the existing config has  `confg/path/to/file` - ie with the `config/` prefix

i opted to leave it that way rather than changing the docs - but i can change it

Reply:
Hum, sorry, my bad. This is right.